doc/dynamo0p3.rst

```diff
@@ -408,22 +458,24 @@ need not be on the same space.
 
 As mentioned earlier, not all combinations of metadata are
 valid. Valid combinations are summarised here. All types of data
-(``GH_INTEGER``, ``GH_REAL``, ``GH_FIELD`` and ``GH_OPERATOR``) may
-be read within a Kernel and this is specified in metadata using
-``GH_READ``. At least one kernel argument must be listed as being
-modified. When data is *modified* in a Kernel then the permitted access
-modes depend on the type of data it is and the function
-space it is on. Valid values are given in the table below.
-
-=============     ============================    =======================
-Argument Type     Function space                  Access type
-=============     ============================    =======================
-GH_INTEGER        n/a                             GH_SUM (Built-ins only)
-GH_REAL           n/a                             GH_SUM (Built-ins only)
-GH_FIELD          Discontinuous (w3)              GH_WRITE
-GH_FIELD          Continuous (not w3)             GH_INC
-GH_OPERATOR       Any for both 'to' and 'from'    GH_WRITE
-=============     ============================    =======================
+(``GH_INTEGER``, ``GH_REAL``, ``GH_FIELD``, ``GH_OPERATOR`` and
+``GH_COLUMNWISE_OPERATOR``) may be read within a Kernel and this is
+specified in metadata using ``GH_READ``. At least one kernel argument
+must be listed as being modified. When data is *modified* in a Kernel
+then the permitted access modes depend on the type of data it is and
+the function space it is on. Valid values are given in the table
+below.
+
+======================	============================    =======================
+Argument Type     	Function space                  Access type
+======================	============================    =======================
+GH_INTEGER        	n/a                             GH_SUM (Built-ins only)
+GH_REAL           	n/a                             GH_SUM (Built-ins only)
+GH_FIELD                Discontinuous (w3)              GH_WRITE
+GH_FIELD                Continuous (not w3)             GH_INC
+GH_OPERATOR             Any for both 'to' and 'from'    GH_WRITE
+GH_COLUMNWISE_OPERATOR  Any for both 'to' and 'from'    GH_WRITE
+======================  ============================    =======================
 
 Note that only Built-ins may modify scalar arguments. There is no
 restriction on the number and function-spaces of other quantities that
```

Comment:
The Subroutine section also needs looking at. Presumably we now have different rules for the different types of Kernels and we have new datatypes that we need to give examples of, in terms of their dimensions. If this information is not yet known then there should at least be placeholders.

Also, do the new kernel types have any different behavior when it comes to parallelization, loop fusion etc. If so, this information needs to be captured.